website/docs/docs/build/python-models.md

```diff
@@ -221,6 +241,29 @@ def model(dbt, session):
 
 </File>
 
+<VersionBlock firstVersion="1.8">
+    
+
+#### Dynamic configurations
+
+In addition to the existing methods of configuring Python models, you also have dynamic access to configuration values set with `dbt.config()` within Python models using f-strings. This expands the possibilities for custom logic and configuration management.
```

Comment:
sugg: i think when using "possibility" , it's generally collocated with "increase"

```suggestion
In addition to the existing methods of configuring Python models, you also have dynamic access to configuration values set with `dbt.config()` within Python models using f-strings. This increases the possibilities for custom logic and configuration management.
```